### ISSUE TITLE
Config value for proxy.config.hostdb.host_file.interval is never read.

### ISSUE THREAD

**Issue by cmcfarlen** [apache/trafficserver#9024]
This config item is for setting how often the host_file is re-read.  It is documented and registered in RecordsConfig, but is never read into the `hostdb_hostfile_check_interval` variable.

**Comment by bneradt** [comment]
We have this internally in Yahoo (I think this didn't make its way to ASF because HostDB was merged just recently):

```
From 71f1c3d1e4d8c57149be0605bd56f0ebf301121e Mon Sep 17 00:00:00 2001
From: "Alan M. Carroll" <amc@apache.org>
Date: Sun, 23 Jan 2022 16:05:07 -0600
Subject: [PATCH] Fix HostDB sync time variable.

(cherry picked from commit 0830c0c368f263fab58f3e051dd3a85e99c3e88c)
---
 iocore/hostdb/HostDB.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/iocore/hostdb/HostDB.cc b/iocore/hostdb/HostDB.cc
index e8124863fe..01f6ee20ef 100644
--- a/iocore/hostdb/HostDB.cc
+++ b/iocore/hostdb/HostDB.cc
@@ -208,12 +208,13 @@ extern MgmtConverter const &HostDBDownServerCacheTimeConv;
 MgmtConverter const &HostDBDownServerCacheTimeConv = HostDBDownServerCacheTimeVar.Conversions;
 
 // Not run time configurable, therefore no support beyond this class needed.
-ConfigDuration HostDBSyncFrequency{hostdb_sync_frequency};
+ConfigDuration HostDBSyncFrequencyVar{hostdb_sync_frequency};
 
 void
 HostDB_Config_Init()
 {
   HostDBDownServerCacheTimeVar.Enable("proxy.config.http.down_server.cache_time");
+  HostDBSyncFrequencyVar.Enable("proxy.config.cache.hostdb.sync_frequency");
 }
 
 // Static configuration information
```

**Comment by ywkaras** [comment]
So `proxy.config.cache.hostdb.sync_frequency` replaces `proxy.config.hostdb.host_file.interval`?

**Comment by bneradt** [comment]
> So `proxy.config.cache.hostdb.sync_frequency` replaces `proxy.config.hostdb.host_file.interval`?

Great observation. @cmcfarlen  fortunately used the right variable in his #9039 PR. That said, I'll pull in the hostdb.sync_frequency patch.

**Comment by bneradt** [comment]
Closing since the fix in PR #9039 is merged.
